x-pack/plugins/maps/public/classes/sources/es_search_source/scaling_form.tsx

```diff
@@ -154,6 +165,33 @@ export class ScalingForm extends Component<Props, State> {
     );
   }
 
+  _renderLimitResultsRadio() {
```

Comment:
Are functions really needed for limit and top hits options? I would keep as inlined html